test/recipes/02-test_list.t

```diff
@@ -0,0 +1,19 @@
+#! /usr/bin/env perl
+# Copyright 2022 The OpenSSL Project Authors. All Rights Reserved.
+#
+# Licensed under the Apache License 2.0 (the "License").  You may not use
+# this file except in compliance with the License.  You can obtain a copy
+# in the file LICENSE in the source distribution or at
+# https://www.openssl.org/source/license.html
+
+use OpenSSL::Test;
+use OpenSSL::Test::Utils;
+
+setup("test_list");
+
+plan skip_all => "No list tests without QUIC"
+    if disabled("quic");
```

Comment:
!

Reply:
Agree 100%, however until it is used outside of QUIC, there seems little point including it there.

Reply:
> however until it is used outside of QUIC, there seems little point including it there. Once it is used, it's trivial to switch.

So now, you're putting in *extra effort* to limit this generic thing to QUIC, and thus ensuring we must put in *extra effort* to remove that limitation later on.
Yeah, we have obviously different ideas on priorities...

Reply:
Currently, it's done this way.  It's extra effort to change it!

Reply:
[shrug]